LanguageExt.Tests/NullChecks/AbstractNullCheckTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿using Xunit;
+
+namespace LanguageExt.Tests.NullChecks
+{
+    public abstract class AbstractNullCheckTests
+    {
+        protected abstract bool ExpectedWhenNull { get; }
+        protected abstract bool NullCheck<T>(T value);
+
+        private bool ExpectedWhenNotNull => !ExpectedWhenNull;
+
+        [Fact]
+        public void NullCheck_NullObject_AsExpectedWhenNull()
+        {
+            object value = null;
+
```

Comment:
This commit is all great, but could you do me a favour and remove the empty-line every other line:

So this:
```c#
object value = null;

var actual = NullCheck(value);

Assert.Equal(ExpectedWhenNull, actual);
```
Becomes:
```c#
object value = null;
var actual = NullCheck(value);
Assert.Equal(ExpectedWhenNull, actual);
```
Just as a general rule I like to keep related blobs of code together, only spacing when there's a natural 'pause' in what's happening - the message to the programmer being "We've done that, now we're doing this".  I think because the setup/test/assert are one-liners here, there's no real "we're doing the setup bit, now we're moving to the testing bit, ..."

Reply:
Sure thing.